assets/src/js/admin/reports/pages/overview-page/index.js

```diff
@@ -3,13 +3,76 @@
 // 12 column grid for content to exist in
 
 import Grid from '../../components/grid'
+import Card from '../../components/card'
```

Comment:
@JasonTheAdams Need your confirmation here. Can we setup a `@givewp/component` workspace for these? because in future make we can take leverage of this like frontend or another screen in backend

Reply:
For sure! I highly recommend do so for reusable presentational components.

Reply:
@henryholtgeerts Check out GiveIO for example of how we're splitting up our code into `app`, `common`, and `component` workspaces.

Reply:
@JasonTheAdams @ravinderk Good thinking! I'll take a look at how you're doing things in GiveIO and get that setup today.